less/mixins/grid.less

```diff
@@ -13,8 +13,10 @@
 
 // Creates a wrapper for a series of columns
 .make-row(@gutter: @grid-gutter-width) {
-  margin-left:  ceil((@gutter / -2));
-  margin-right: floor((@gutter / -2));
+  @_margin_left_helpvar:  ceil((@gutter / 2));
```

Comment:
I had to use the "helpvars" to get it to work 😞 
E.g. having `margin-left: -ceil((@gutter / 2));` in Less outputs `margin-left: -ceil(7.5px);` directly into the CSS (and not `-8px`).
I was unable to find a way of writing it that Less understood – without using the extra variables.